#
# Submitted-By: Robert Schwebel, 2006-10-20
# Committed-By: Robert Schwebel
#
# Error:
#
# inputmethod/qwsinputcontext_qws.cpp:183: error: 'qt_sendSpontaneousEvent' was not declared in this scope 
#
# Description:
#
# declaration seems to be missing
#
# State:
#
# should be sent upstream
#

diff -urN qtopia-core-opensource-src-4.1.1/src/gui/inputmethod/qwsinputcontext_qws.cpp qtopia-core-opensource-src-4.1.1-ptx/src/gui/inputmethod/qwsinputcontext_qws.cpp
--- qtopia-core-opensource-src-4.1.1/src/gui/inputmethod/qwsinputcontext_qws.cpp	2006-02-15 14:03:36.000000000 +0100
+++ qtopia-core-opensource-src-4.1.1-ptx/src/gui/inputmethod/qwsinputcontext_qws.cpp	2006-10-20 15:11:54.000000000 +0200
@@ -35,6 +35,8 @@
 
 #ifndef QT_NO_QWS_INPUTMETHODS
 
+extern bool qt_sendSpontaneousEvent(QObject *, QEvent *);
+
 static QWidget* activeWidget = 0;
 
 //#define EXTRA_DEBUG
